README.md

```diff
@@ -205,17 +203,11 @@ Example:
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+👤 **Muzaffer Dler**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@Muzaffer Dler](https://github.com/jellywiz/)
+- LinkedIn: [@Muzaffer Dler](https://www.linkedin.com/in/muzaffer-dler-473484205/)
+- LinkedIn: [@Muzaffer Dler](https://www.linkedin.com/in/muzaffer-dler-473484205/)
```

Comment:
@Gambit142 I am not sure what would be my future features so for now i will delete them since i have nothing in mind for the future to be added

Reply:
@jellywiz Kindly implement the requirement of adding the Future Features sections with 1-3 features as earlier stated. It is essential because most people that would like to contribute to the project would easily find features to add or work on. Since you are building a simple calculator, you could mention adding a scientific calculator as a feature. Kindly use Google to check more features that can be added to the project. 👍